poly/constants.go

```diff
@@ -0,0 +1,15 @@
+package poly
+
+import (
+	"github.com/dedis/crypto/edwards"
+)
+
+const (
```

Comment:
I see.  But then there should be parameterized configuration state variables that actually describe what they do, rather than global constants with vague names like "minimum/moderate/maximum security." :)

Also, now might actually be a good time to do away with that whole "Certified" process and threshold once and for all anyway.  Any roadblocks to doing that?  Or is that more tricky than it seems for any reason, which might warrant doing it separately?